sieves/tasks/predictive/ner/bridges.py

```diff
@@ -0,0 +1,493 @@
+
+
+class GliXNER(NERBridge[list[str], glix_.Result, glix_.InferenceMode]):
+    def __init__(
+        self,
+        entities: list[str],
+        task_id: str,
+        prompt_template: str | None,
+        prompt_signature_desc: str | None,
+    ):
+        """
+        Initializes GliXNER bridge.
+        :param entities: List of entity types to extract.
+        :param task_id: Task ID.
+        :param prompt_template: Custom prompt template.
+        :param prompt_signature_desc: Custom prompt signature description.
+        """
+        super().__init__(
+            entities=entities,
+            task_id=task_id,
+            prompt_template=prompt_template,
+            prompt_signature_desc=prompt_signature_desc,
+        )
+
+    @property
+    def _prompt_template(self) -> str | None:
+        return None
+
+    @property
+    def _prompt_signature_description(self) -> str | None:
+        return None
+
+    @property
+    def prompt_signature(self) -> list[str]:
+        return self._entities
+
+    @property
+    def inference_mode(self) -> glix_.InferenceMode:
+        return glix_.InferenceMode.ner
+
+    def consolidate(
+        self, results: Iterable[glix_.Result], docs_offsets: list[tuple[int, int]]
+    ) -> Iterable[glix_.Result]:
+        results = list(results)
+
+        # Simply group results by document without trying to adjust positions
+        # Position adjustment will happen in the integrate function
+        for doc_offset in docs_offsets:
+            doc_results = results[doc_offset[0] : doc_offset[1]]
+            all_entities: list[dict[str, Any]] = []
+
+            # Keep track of which chunk each entity came from
+            for chunk_idx, chunk_result in enumerate(doc_results):
+                # Process entities in this chunk
+                for entity in chunk_result:
+                    if isinstance(entity, dict):
+                        # Add chunk index to the entity for reference in integrate
+                        entity_copy = entity.copy()
+                        entity_copy["chunk_idx"] = chunk_idx
+                        all_entities.append(entity_copy)
+
+            # Yield results for this document (flattened list of entities)
+            yield all_entities
+
+    def integrate(self, results: Iterable[glix_.Result], docs: Iterable[Doc]) -> Iterable[Doc]:
+        docs_list = list(docs)
+        results_list = list(results)
+
+        class Entity(pydantic.BaseModel):
+            text: str
+            start: int
+            end: int
+            entity_type: str
+
+        class Entities(pydantic.BaseModel):
+            text: str
+            entities: list[Entity] = []
+
+        # Process each document
+        for doc, result in zip(docs_list, results_list):
+            entities_list: list[Entity] = []
+            doc_text = doc.text if doc.text is not None else ""
+
+            # Get chunk information from the document
+            chunk_offsets = []
```

Comment:
Always type empty collections.
```suggestion
            chunk_offsets: list[str] = []
```